kafka/consume.go

```diff
@@ -192,6 +240,41 @@ func (r *ConfluentConsumer) Close() error {
 	return r.c.Close()
 }
 
+// GetMeta return the confluence consumer metatdata
+func (r *ConfluentConsumer) GetMeta(allTopics bool, timeout time.Duration) (*kafkalib.Metadata, error) {
```

Comment:
What do you think about setting a default timeout in the Consumer for operations, then having some kind of override? Instead of having it in each signature? 

I think that it might be a way to get rid of the boilerplate checks

Reply:
I actually flip flopped on this. I had it as a default setting in my last commit and then changed it to follow the other signatures in the library. I don't have any strong feelings either way￼

Reply:
I think a default is my light preference.

Reply:
Also, can we change the name of the method to `GetMetadata` to be more friendly?

Reply:
What do you think would be a good default timeout, 30sec?  I will make it an option in the config?

Reply:
Yeah I'd make it an option in the config, but default of 30s is plenty imho